server/src/service/CardService.js

```diff
@@ -10,4 +11,22 @@ export class CardService extends BaseService {
 
         return CardService.instance;
     }
+
+    @Transactional()
+    async getCardCountByPeriod({ startDate, endDate, boardIds, userId }) {
+        let query = this.cardRepository
+            .createQueryBuilder('card')
+            .select(`date_format(card.due_date, '%Y-%m-%d')`, 'dueDate')
+            .addSelect('count(1)', 'count')
+            .innerJoin('card.list', 'list', 'list.board_id IN(:boardIds)', { boardIds })
```

Comment:
```suggestion
            .innerJoin('card.list', 'list', 'list.board_id IN(:...boardIds)', { boardIds })
```
https://typeorm.io/#/select-query-builder/using-parameters-to-escape-data
위 문서 보면 `...` 을 붙여줘야 하는걸로 나오는데, 확인해보셨나요?

Reply:
cardRouter 테스트 코드 보면은 내가 만든 보드와 초대 받은 보드를 give로 하나씩 생성하고 테스트를 진행하고 있습니다.
두개의 보드가 있으니 boardIds가 [boardId1, boardId2]로 나올 것으로 예상했었습니다. 
테스트가 정상 통과가 되서 세퍼레이터를 안써도 문제는 없는 듯했는데 공식문서에서 안내해주는대로 사용하는게 좋을거 같네요.
수정하겠습니다:)